packages/shorebird_cli/lib/src/commands/account/account_usage_command.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:async';
+
+import 'package:barbecue/barbecue.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_validation_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template account_usage_command}
+/// `shorebird account usage`
+/// Get usage information for your Shorebird account.
+/// {@endtemplate}
+class AccountUsageCommand extends ShorebirdCommand
+    with ShorebirdConfigMixin, ShorebirdValidationMixin {
+  @override
+  String get description => 'Get usage information for your Shorebird account.';
+
+  @override
+  String get name => 'usage';
+
+  @override
+  Future<int> run() async {
+    try {
+      await validatePreconditions(checkUserIsAuthenticated: true);
```

Comment:
We probably want `checkShorebirdInitialized` here too

Reply:
why? this is an account level command so I don't think it needs to be run within a shorebird app

Reply:
Ah you're right, I was thinking this would be getting app id from shorebird.yaml, but nope!